ocs_ci/utility/openshift_dedicated.py

```diff
@@ -94,15 +112,28 @@ def get_kubeconfig(cluster, path):
     path = os.path.expanduser(path)
     basepath = os.path.dirname(path)
     os.makedirs(basepath, exist_ok=True)
-    cluster_details = get_cluster_details(cluster)
-    cluster_id = cluster_details.get("id")
-    cmd = f"ocm get /api/clusters_mgmt/v1/clusters/{cluster_id}/credentials"
-    out = run_cmd(cmd)
-    credentials = json.loads(out)
+    credentials = get_credentials(cluster)
     with open(path, "w+") as fd:
         fd.write(credentials.get("kubeconfig"))
 
 
+def get_kubeadmin_password(cluster, path):
```

Comment:
Name of this function should be changed to something like `store_kubeadmin_password`. `get_kubeadmin_password` suggests that the password is obtained as return value of the function.

Reply:
get_kubeconfig and get_kubeadmin_password for storing instead of getting this data are used in other parts of our project as well: https://github.com/red-hat-storage/ocs-ci/blob/f29e45d375336e09dd178e4820a9949b0784725c/ocs_ci/utility/utils.py#L2422
https://github.com/red-hat-storage/ocs-ci/blob/f29e45d375336e09dd178e4820a9949b0784725c/ocs_ci/utility/openshift_dedicated.py#L85
https://github.com/red-hat-storage/ocs-ci/blob/f29e45d375336e09dd178e4820a9949b0784725c/ocs_ci/utility/ibmcloud.py#L188
I've opened an issue https://github.com/red-hat-storage/ocs-ci/issues/5158 for renaming them consistently.

Reply:
Ok, thanks